# -------------------------------------------
#   Waypoint Planner -- parameters
# -------------------------------------------

#search_radius             : 50
#search_fov                : 120
search_radius             : 75
search_fov                : 360
local_pose_topic          : aidtr_auto/local_pose
#autonomy_server_topic     : /nrec/zone_recon_autonomy
autonomy_server_topic     : autonomy_server
search_polygon_topic      : /search_polygon/specification
search_polygon_viz_topic  : search_polygon/visualisation
costmap_update_topic      : map
viz_costmap_topic         : visibility/map
viz_costmap_topic_lowres  : visibility/map_lowres
map_frame                 : /map_zero
start_frame               : /start_zero
odom_frame                : /odom_zero
track_topic               : tracks
search_mode               : nats
robot_type                : ugv
large_map_flag            : true # controls whether to use the ompl planner or not
goal_output_frame         : /map_zero
all_rviz                  : false